_aicspylibczi/Reader.cpp

```diff
@@ -206,18 +209,11 @@ Reader::getAllSceneYXSize(int scene_index_, bool get_all_matches_)
 {
   std::vector<libCZI::IntRect> result;
   bool hasScene = m_statistics.dimBounds.IsValid(libCZI::DimensionIndex::S);
-  if (!isMosaic() && hasScene) {
-    int sStart(0), sSize(0);
-    m_statistics.dimBounds.TryGetInterval(libCZI::DimensionIndex::S, &sStart, &sSize);
-    if (scene_index_ >= sStart && (sStart + sSize - 1) >= scene_index_ && !m_statistics.sceneBoundingBoxes.empty()) {
-      result.emplace_back(m_statistics.sceneBoundingBoxes[scene_index_].boundingBoxLayer0);
-      return result;
-    }
-  }
```

Comment:
this method which uses boundingBoxLayer0 can return a different result than the following code which uses the subblock logicalRects.  Preferring the logicalRect gives consistency between expected shape and read shape

Reply:
Hi Dan, I've reviewed the changes you've made and agree with them.